polling_stations/apps/data_collection/data_quality_report.py

```diff
@@ -0,0 +1,374 @@
+from django.db import connection
+from django.db.models import Q
+from pollingstations.models import PollingStation, PollingDistrict, ResidentialAddress
+
+
+# define some methods we can use to print coloured console output
+class OutputFormatter:
+    OKGREEN = '\033[92m'
```

Comment:
It might be better to use https://docs.python.org/3/library/curses.html for colour stuff – I think it can get quite complex…